go/vt/topo/test/replication.go

```diff
@@ -27,9 +27,10 @@ import (
 
 // checkShardReplication tests ShardReplication objects
 func checkShardReplication(t *testing.T, ts topo.Impl) {
```

Comment:
It feels like this function needs to accept Server instead of Impl as well.

Reply:
Done.